channels/serializers.py

```diff
@@ -772,7 +779,11 @@ def create(self, validated_data):
         else:
             raise ValueError("Missing moderator_name or email")
 
-        return api.add_moderator(username, channel_name)
+        api.add_moderator(username, channel_name)
+        return api._list_moderators(
```

Comment:
Lint is complaining about this line.  Maybe a `get_moderator(username, channel_name)` function could be created and used here, or the `add_moderator` function could be reverted to return the Redditor object again, or just add a `pylint:disable` line.  Whichever you think is best is fine by me.